audio-analysis/main.py

```diff
@@ -1,10 +1,40 @@
 import videoHelper
+import pandas as pd
 
-trumpVideo = videoHelper.videoObject('https://www.youtube.com/watch?v=JZRXESV3R74', False)
-print(trumpVideo.getFilename())
-trumpVideo.getAudio()
-trumpVideo.getTextAnalysis()
-trumpVideo.getAmplitudeAnalysis()
+# Save Video function creates a video object and outputs all the nice analysis stuff
+# Will add a function to consolidate video data into a dictionary 
+# Will add a function to push video dictionaries to SQL table
+def saveVideo(ytURL, hTag):
+    videoX = videoHelper.videoObject(ytURL, hTag)
+    print(videoX.getFilename())
+    videoX.getAudio()
+    videoX.getTextAnalysis()
+    videoX.getAmplitudeAnalysis()
+
+##### Prompt for Highlight Video and Tagging #####
+answer = input("Are you entering a single video (enter 'single') or CSV (enter 'csv') \n")
```

Comment:
how about smth like option 1. single 2. csv so we input 1 or 2 versus single and csv

Reply:
... and a deafult behaviour so we can just skip all and it will run a test on the OG trump video